### PR TITLE
Translate tutorial 4_new_modules

### PR DESCRIPTION
**Pull request by zengwang430521**

### CODE REVIEW

**Review by jin-s13 — APPROVED** [review]

File: docs_zh-CN/tutorials/4_new_modules.md [hunk 1]
```diff
@@ -1,3 +1,214 @@
 # 教程 4: 增加新的模块
 
-内容建设中……
+## 自定义优化器
+
+在本教程中，我们将介绍如何为项目定制优化器.
+假设用户想要添加一个名为 `MyOptimizer` 的优化器，它有 `a`，`b` 和 `c` 三个参数。
+那么用户首先需要在一个文件中实现该优化器，例如 `mmpose/core/optimizer/my_optimizer.py`：
```
Comment (innerlee) [inline docs_zh-CN/tutorials/4_new_modules.md:7]: ~~用户~~

File: docs_zh-CN/tutorials/4_new_modules.md [hunk 2]
```diff
@@ -1,3 +1,214 @@
...
+
+
+@OPTIMIZERS.register_module()
+class MyOptimizer(Optimizer):
+
+    def __init__(self, a, b, c)
+
+```
+
+然后用户需要将其添加到 `mmpose/core/optimizer/__init__.py` 中，从而让注册器可以找到这个新的优化器并添加它：
```
Comment (innerlee) [inline docs_zh-CN/tutorials/4_new_modules.md:21]: ~~用户~~

File: docs_zh-CN/tutorials/4_new_modules.md [hunk 3]
```diff
@@ -1,3 +1,214 @@
...
+
+```python
+from .my_optimizer import MyOptimizer
+```
+
+之后，用户便可以在配置文件的 `optimizer` 字段中使用 `MyOptimizer`。
```
Comment (innerlee) [inline docs_zh-CN/tutorials/4_new_modules.md:27]: ~~用户~~

File: docs_zh-CN/tutorials/4_new_modules.md [hunk 4]
```diff
@@ -1,3 +1,214 @@
...
+
+用户可以直接根据 [PyTorch API 文档](https://pytorch.org/docs/stable/optim.html?highlight=optim#module-torch.optim)
+对参数进行设置。
+
+## 自定义优化器构造器
+
+某些模型可能对不同层的参数有特定的优化设置，例如 BatchNorm 层的权值衰减。
+用户可以通过自定义优化器构造函数来进行这些细粒度的参数调整。
+
+```
```
Comment (innerlee) [inline docs_zh-CN/tutorials/4_new_modules.md:56]: ` ```python `

File: docs_zh-CN/tutorials/4_new_modules.md (hunk 2 above)
Comment (zengwang430521) [reply docs_zh-CN/tutorials/4_new_modules.md]: Fixed.

**Review by ly015 — APPROVED** [review]
